src/cli/README.md

```diff
@@ -1193,29 +1193,42 @@ Done
 ### prefix
 
 Get the prefix list in the local Network Data.
+Note: For the border router with backbone capability, the local Domain Prefix
+      would be listed as well (with flag `D`) regardless of that backone
+      functionality is enabled.
 
 ```bash
 > prefix
 2001:dead:beef:cafe::/64 paros med
 Done
 ```
 
-### prefix add \<prefix\> [pvdcsr] [prf]
+### prefix add \<prefix\> [padcrosnD] [prf]
 
 Add a valid prefix to the Network Data.
 
+Note: The Domain Prefix flag(`D`) is only available for Thread 1.2.
+      If the domain prefix is configured to the local backbone router server.
+      And it would not be added to local server data unless backbone router
+      feature is enabled.
```

Comment:
I don't think these three lines are necessary. Implementation below shows that CLI will return error if `D` flag is specified.

Reply:
removed.